import-export-cli/impl/importAPI.go

```diff
@@ -257,13 +257,13 @@ func ImportAPI(accessOAuthToken, publisherEndpoint, importEnvironment, importPat
 		return err
 	}
 
-	utils.Logln(utils.LogPrefixInfo + "Attempting to inject parameters to the API from api_params.yaml (if exists)")
+	utils.Logln(utils.LogPrefixInfo + "Attempting to process environment configurations directory or file")
```

Comment:
Does this print any kind of separator between utils.LogPrefixInfo prefix and the rest of the text? i.e. ":"

Reply:
In the **utils.LogPrefixInfo** constant is defined with this ":" character as the separator.  The value of this constant looks like this [1].
[1]. https://pkg.go.dev/github.com/wso2/product-apim-tooling/import-export-cli/utils#LogPrefixInfo